src/main/java/eu/ai4work/sws/model/generic/SlidingDecisionRequest.java

```diff
@@ -0,0 +1,11 @@
+package eu.ai4work.sws.model.generic;
+
+import lombok.Data;
+import java.util.Map;
+
+@Data
+public class SlidingDecisionRequest {
+    private String decisionStatus;
```

Comment:
I think we can remove this field, it does not really have any added value. It was originally just added to make the input JSON easier to read/understand, but I think it is also understandable without.

Reply:
This field is removed in the commit https://github.com/AI4WORK-Project/sliding-work-sharing/pull/2/commits/6e63b8bce37097860fbe069304276275bb28ba83